bundles/remove.rst

```diff
@@ -60,18 +53,13 @@ quickly spot bundle configuration by looking for an ``acme_demo`` (or whatever
 the name of the bundle is, e.g. ``fos_user`` for the FOSUserBundle) string in
 the configuration files.
 
-The AcmeDemoBundle doesn't have configuration. However, the bundle is
-used in the configuration for the ``app/config/security.yml`` file. You can
-use it as a boilerplate for your own security, but you **can** also remove
-everything: it doesn't matter to Symfony if you remove it or not.
-
 3. Remove the Bundle from the Filesystem
 ----------------------------------------
 
 Now you have removed every reference to the bundle in your application, you
-should remove the bundle from the filesystem. The bundle is located in the
-``src/Acme/DemoBundle`` directory. You should remove this directory and you
-can remove the ``Acme`` directory as well.
+should remove the bundle from the filesystem. The bundle will be located in within
```

Comment:
Small typo; `in within` is double. Suggestion; `will be located in src/`.
Also `src/` needs double backticks around it.